lib/param_parsing/versiontf.go

```diff
@@ -0,0 +1,34 @@
+package param_parsing
+
+import (
+	"github.com/hashicorp/terraform-config-inspect/tfconfig"
+	"github.com/warrensbox/terraform-switcher/lib"
+	"os"
+)
+
+const versionTfFileName = "version.tf"
+
+func GetVersionFromVersionsTF(params Params) Params {
+	filePath := params.ChDirPath + "/" + versionTfFileName
```

Comment:
It makes no sense as TF version is read from the **module** (from the directory) rather than from the specific file.
Please re-work this package to rely upon the directory (the `params.ChDirPath`) instead of any particular file in it.
Thank you.

Reply:
Done